models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1598 @@
+   5.44649167346675
+   5.45489142211957
+   5.45808308572015
+   3.479253215076174E-002
+   3.469211455745640E-002
+   5.41512442472872
+   5.41843086313655
+   5.44649167346675
+   5.44970758027390
+   5.50877061923831
+   5.51180677764943
+   5.46367193547511
+   5.46683825691274
+   5.44039845768897
+   5.44363195062815
+   0.000000000000000E+000
+   0.000000000000000E+000
+           2           4          -1
+ obdef
+ loc3d
+     5.235987755982989         0.000000000000000        -888888.0000000000     -2
+ kind
+          23
+     0     149750
+  0.200000000000000
+
+Below we provide the variable names for the ``clm_obs_seq.2011-01-02-00000.final``
+example from above.
+
+::
+
+   <observation sequence number>
+   <observation value>
+   <true observation value>
+   <prior ensemble mean>
+   <posterior ensemble mean>
+   <prior ensemble spread>
+   <posterior ensemble spread>
+   <prior member 1>
+   <posterior member 1>
+   <prior member 2>
+   <posterior member 2>
+   <prior member 3>
+   <posterior member 3>
+   <prior member 4>
+   <posterior member 4>
+   <prior member 5>
+   <posterior member 5>
+   <data product QC>
+   <DART quality control>
+ 
+ obdef
+ loc3d
+     <longitude>    <latitude>   <vertical level>   <vertical code>
+ kind
+    <observation quantity number>
+    <seconds>     <days>
+    <observation error variance>
+
+
+In the example above, the observation has been **accepted**
+denoted by a ``DART quality control value = 0``. If the 
+``DART quality control value =7`` this indicates the observation has
+fallen outside the ``outlier_threshold`` value and is rejected. 
+For more details on the DART quality control variables read the  
+:doc:`documentation. <../../../assimilation_code/modules/assimilation/quality_control_mod>`
+
+First, in your own tutorial assimilation confirm that this observation
+(and other observations) was accepted.
+
+Second, confirm that the ``posterior member`` values have been updated
+from their respective ``prior member`` values.  In general the
+``posterior member`` and ``posterior ensemble mean`` values should become
+closer to the `observation value` as compared to the prior values.
+
+
+::
+
+  cd <caseroot>
+  less clm_obs_seq.2011-01-02-00000.final
+
+
+This tutorial has been purposely designed such that all observations
+are accepted and the posteriors have been updated.  In research applications,
+however, the vast majority of observations may be rejected if there is
+large systemic biases between the model ensemble and the observations.
+In that case, it may take many assimilation time steps before the inflation
+creates a sufficient enough ensemble spread such that the observation falls
+within the outlier threshold and is accepted.  In other cases, 
+an observation may be accepted, but the posterior update is negligible. 
+If you experience these issues, a helpful troubleshooting guide is 
+located :doc:`here. <../../../guide/dart-quality-control>`
+
+
+
+Matlab Diagnostics
```

Comment:
@braczka, I think it would be useful to actually show a figure or 2 here. For instance, the result of `plot_rmse_xxx_evolution` and `link_obs.m` The user will them compare theirs to the ones in the tutorial.